### DISCUSSION TITLE
Using Jupyter notebook with xeus-sqlite kernel in Visual Studio Code

### DISCUSSION THREAD

**Discussion by bucweat** [microsoft/vscode-jupyter#11236] [Questions and Answers]
Trying to xeus-sqlite in a Jupyter notebook in Visual Studio Code so that I can execute sqlite SQL queries directly. So far I've not been able to see/select the xsqlite kernel in Visual Studio Code. I've searched google and here and, while there are lots of items here regarding loading of kernels, I've not found anything that helps (yet). 

Running Macbook Pro with latest MacOS. Conda is latest and greatest and just did the xeus-sqlite install today. I've installed using conda to a virtual environment with name 'sqlite' per the xeus-sqlite instructions at https://xeus-sqlite.readthedocs.io/en/latest/installation.html.

```
conda create -n sqlite
conda activate sqlite
conda install xeus-sqlite notebook -c conda-forge
jupyter notebook .
```

I create a new notebook, and xsqlite kernel is available and can be selected and xeus-sqlite magic and SQL commands (e.g. %LOAD magic) work as expected...

![notebook_kernels](https://user-images.githubusercontent.com/3979338/187668203-8d267454-310c-40b1-a984-e3d9efc233c3.png)

I verified that xsqlite shows up with `jupyter kernelspec list`:

<img width="576" alt="terminal_kernels" src="https://user-images.githubusercontent.com/3979338/187668534-4f544d33-3563-47a8-8205-38a0e2518c06.png">

Here is what my kernel list looks like in Visual Studio Code:

<img width="638" alt="vscode_kernels" src="https://user-images.githubusercontent.com/3979338/187668894-f47d0ce4-babd-4728-a1d4-c7574507cef9.png">

As you can see, the conda sqlite virtual env shows up, but only the python kernel is present and not the xsqlite kernel. I do see that the path to the kernel in Visual Studio Code is different from that reported by `jupyter kernelspec list` so maybe that is the issue...just not sure how to correctly configure that if so...

This is a pretty niche requirement and so not much found with google-fu. I'm hoping someone here has been down this path already...

**Comment by DonJayamanne** [comment]
I've created an issue https://github.com/microsoft/vscode-jupyter/issues/11240 so we can discuss this bug.
